### PR TITLE
fix no-duplicate-string rule with imports

### LINKED ISSUE #700: no-duplicate-string false positive on imports
**I want to report a bug.**

**Rule key:** no-duplicate-string

**Reproducer** 
```
 // False-positive: tslint: Define a constant instead of duplicating this literal 8 times.
import {a} from './very_looooooooooooooooooooooooooooooooooong_name';
import {b} from './very_looooooooooooooooooooooooooooooooooong_name';
import {c} from './very_looooooooooooooooooooooooooooooooooong_name';
```

**Expected behavior**
No warning

### PR DESCRIPTION
**Pull request by maximelkin**
Fixes #700

### CODE REVIEW

File: sonarts-core/tests/rules/noDuplicateStringRule/noDuplicateStringRule.lint.tsx [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+import defaultExport11, * as name from "module-name-long";
+import defaultExport12, * as name from "module-name-long";
+import defaultExport13, * as name from "module-name-long";
+
+import "module-name-long";
+import "module-name-long";
+import "module-name-long";
+
+import a1 = require("module-name-long")
+import a1 = require("module-name-long")
```
Comment (vilchik-elena) [inline sonarts-core/tests/rules/noDuplicateStringRule/noDuplicateStringRule.lint.tsx]: `a2`?

File: sonarts-core/tests/rules/noDuplicateStringRule/noDuplicateStringRule.lint.tsx [hunk 2]
```diff
@@ -1,3 +1,53 @@
+
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+
+// taken from
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import
+import defaultExport from "module-name-long";
```
Comment (vilchik-elena) [inline sonarts-core/tests/rules/noDuplicateStringRule/noDuplicateStringRule.lint.tsx]: @maximelkin do we really need such exhaustive testing of all import and export variants? I would be ok, for couple import variants and couple export.

**Comment by maximelkin** [comment]
Ok, i fix it later

**Comment by maximelkin** [comment]
@vilchik-elena fixed

**Comment by vilchik-elena** [comment]
Replacing this PR with https://github.com/SonarSource/SonarTS/pull/709 (opened on branch of original repo to have full QA pipe)

**Pull request closed without merging**